charts/linkerd-control-plane/templates/destination-rbac.yaml

```diff
@@ -165,7 +165,9 @@ webhooks:
   - operations: ["CREATE", "UPDATE"]
     apiGroups: ["policy.linkerd.io"]
     apiVersions: ["v1alpha1", "v1beta1"]
-    resources: ["servers"]
+    resources:
+    - serverauthorizations
+    - servers
```

Comment:
In at least this case: no. If anything we should probably move some of the others to use explicit lists. My reasoning is that:

1. we're going to add more soon, and this line is going to get very long.
2. this helps minimize diffs as there are changes (lines are inserted/removed, instead of the whole line changing).